libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/factories/NotificationCreator.kt

```diff
@@ -178,7 +180,11 @@ class DefaultNotificationCreator @Inject constructor(
             // 'importance' which is set in the NotificationChannel. The integers representing
             // 'priority' are different from 'importance', so make sure you don't mix them.
             .apply {
-                if (roomInfo.shouldBing) {
+                val notificationManager = NotificationManagerCompat.from(context)
+                val previousNotification = notificationManager.activeNotifications.find { it.tag == roomInfo.roomId.value }
+                val elapsed = System.currentTimeMillis() - (previousNotification?.postTime ?: 0L)
+                Timber.d("Creating noisy notification for room ${roomInfo.roomId.value} with elapsed time $elapsed")
```

Comment:
Move this log into the if block (not sure)

Reply:
Ooops, this should actually be gone. It was done for an experiment and I removed part of it, but apparently I kept this by mistake.